LayoutTests/editing/pasteboard/data-transfer-get-data-on-paste-skipped-content.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<body>
+<script src="../../resources/js-test.js"></script>
+<div id="container">
+<button onclick="runTest()">Copy</button>
+<div id="source" contenteditable="true">hello, <b>world</b><br>WebKit<span style="content-visibility: hidden;">super sensitive content</span></div>
+<div id="destination" onpaste="check(event)" contenteditable="true" style="width: 500px; height: 100px; border: solid red 1px"></div>
+</div>
+</body>
+<script>
+description('Tests that skipped content is successfully removed from pasted content. To manually test, press "Copy" below and paste (Command+V on macOS and Control+V elsewhere).');
+jsTestIsAsync = true;
+
+function runTest()
+{
+    if (window.internals)
+        internals.settings.setCustomPasteboardDataEnabled(true);
+
+    document.getElementById("source").focus();
+    document.execCommand("SelectAll");
+    document.execCommand("Copy");
+
+    document.getElementById("destination").focus();
+    if (window.testRunner)
+        document.execCommand("PasteAndMatchStyle");
+}
+
+function check(event)
+{
+    shouldBeEqualToString('JSON.stringify(event.clipboardData.types)', '["text/plain"]');
+    shouldBeEqualToString('event.clipboardData.getData("text/plain")', 'hello, world\nWebKit');
+    shouldBeEqualToString('event.clipboardData.getData("text/html")', '');
```

Comment:
Why is text/html empty? That doesn't sound right at all.

Reply:
This is a literal copy paste of `LayoutTests/editing/pasteboard/data-transfer-get-data-on-paste-as-plain-text.html`. I'm not sure what's going on there to be honest. Is it maybe related to `internals.settings.setCustomPasteboardDataEnabled(true);` ?